app/forms/idv/ssn_form.rb

```diff
@@ -1,12 +1,16 @@
 module Idv
   class SsnForm
     include ActiveModel::Model
-    include FormSsnValidator
-
     ATTRIBUTES = [:ssn].freeze
 
     attr_accessor :ssn
 
+    validates :ssn, presence: true
```

Comment:
The profile step actually is not part of the doc auth flow (long story). We'll actually want the SSN unique bit in the `extra` attributes of the form response coming out of this form.

Reply:
I *think* I got that in 9a1b1b48a13c3e34d345030a713e0789b186ea46

Reply:
Looks like it, yeah